src/command_modules/azure-cli-acr/README.rst

```diff
@@ -86,39 +86,39 @@ Update a container registry
         az acr update: Update a container registry.
 
     Arguments
-        --name -n [Required]: Name of container registry.
-        --disable-admin     : Disable admin user.
-        --enable-admin      : Enable admin user.
-        --resource-group -g : Name of resource group.
-        --tags              : Space separated tags in 'key[=value]' format. Use "" to clear existing
-                            tags.
-        --tenant-id -t      : Tenant id for service principal login. Warning: Changing tenant id will
-                            invalidate assigned access of existing service principals.
+        --name -n      [Required]: Name of container registry.
+        --disable-admin          : Disable admin user.
+        --enable-admin           : Enable admin user.
```

Comment:
I've opened #1201 & #1202  to cover these 2 separately. 
This PR is for the model update and if we are ok with that we would like to unblock the new RP roll out which is a part of the swagger update which we need to have today. Otherwise our schedule slips by about a week since the connect folks won't have time utilize the new models. Basically this PR should be able out the model update. 

We will send out a PR for the other 2 items once @djyou is back. 
@tjprescott - does that sound reasonable?